libvast/src/system/archive.cpp

```diff
@@ -94,6 +105,11 @@ archive(archive_type::stateful_pointer<archive_state> self,
         }
       );
     },
+    [=](exporter_atom, const actor& exporter) {
+      auto sender_addr = self->current_sender()->address();
+      self->state.active_exporters.insert(sender_addr);
+      self->monitor<caf::message_priority::high>(exporter);
```

Comment:
Just a question for clarification: is `monitor` always normal priority by default?

Reply:
`monitor` gained the priority parameter so it could be used for this task specifically. As far as I know the default is `normal`.